functions.R

```diff
@@ -49,7 +49,7 @@ get_survey_data <- function() {
                               trim_values = F,
                               use.missings = T)
         
-        if(grepl('gss|piaac|cfcs|sduhs', temp_data)) {
+        if(grepl('gss|piaac|cfcs', temp_data)) {
```

Comment:
@benmbrew Why are we removing sduhs?

Reply:
Sorry, I should have said. I discovered later that the year variable is actually in there. I missed it the first time because it was coded weird, so no need to add year from name.

Reply:
I don't understand your comment. What does "no need to add year from name" mean?

Reply:
I mean we dont need to extract year for that dataset. We get the year from the name of the data set, if that condition is true (get_year).

Reply:
@benmbrew Understood. Thanks.